middle_end/flambda2/simplify/simplify_apply_expr.ml

```diff
@@ -379,7 +371,7 @@ let simplify_direct_partial_application ~simplify_expr dacc apply
           ~result_arity ~stub:true ~inline:Default_inline ~is_a_functor:false
           ~recursive ~cost_metrics:cost_metrics_of_body
           ~inlining_arguments:(DE.inlining_arguments (DA.denv dacc))
-          ~dbg ~is_tupled:false
+          ~dbg ~is_tupled:false ~inlining_decision:Stub
```

Comment:
Should Code.create verify that the decision and the attributes correspond ?

Reply:
What is the end goal of verifying that the decision and the attributes correspond in [Code.create]? I can understand why it might be done in [Code.invariants] but it's less clear to me why one would like to check that in [Code.create].
I believe the inliner will already create a warning something annotated to be inline is actually not.